Update environment-variables.md (#285)
node-red/node-red.github.io: docs/user-guide/environment-variables.md

```javascript
let foo = env.get("FOO");
```
### Template node

> Beginning with [version 3.0](https://nodered.org/blog/2022/07/14/version-3-0-released#environment-variables-in-the-template-node
> ), the `template` node can access environment variables using the syntax:

Reviewer: ```suggestion
*Since [Node-RED 3.0](https://nodered.org/blog/2022/07/14/version-3-0-released#environment-variables-in-the-template-node)*

The `template` node  can access environment variables using the syntax:
```

a subflow could be created to access the API and handle the response, using an
environment variable to identify which type of record should be accessed. Individual
instances of the Subflow can then be customised for those particular types.

### Flow/Group level environment variables

> Since [version 2.1](https://nodered.org/blog/2021/10/21/version-2-1-released#flowgroup-level-environment-variables), environment variables can be set at the flow or group level. This is done in the appropriate tab in the edit dialog for the flow or group.

Reviewer: ```suggestion
*Since [Node-RED 2.1](https://nodered.org/blog/2021/10/21/version-2-1-released#flowgroup-level-environment-variables)*

Environment variables can be set at the flow or group level. This is done in the appropriate tab in the edit dialog for the flow or group.
```

> ### Raspberry Pi OS considerations

Reviewer 2: While the script is mainly aimed at Pi users, it works well for any Debian based install eg Ubuntu. So this section may need renaming.

> ### Raspberry Pi OS considerations

Reviewer: ```suggestion
### Running as a service
```

> Users who have installed Node-RED on a Raspberry Pi using the provided [script](https://nodered.org/docs/getting-started/raspberrypi) and who run it as a service on booting or by using the `node-red-start` command should be aware that their flows will not have access to environment variables that are defined only in the calling process. In this case, environment variables can be defined in the settings file by statements of the form 

Reviewer: ```suggestion
When Node-RED is running as a service having been installed using the provided [script](https://nodered.org/docs/getting-started/raspberrypi), it will not have access to environment variables that are defined only in the calling process. In this instance, environment variables can be defined in the settings file by adding:
```
